src/configmanager.cpp

```diff
@@ -2772,16 +2772,20 @@ void ConfigManager::loadTranslations(QString locale)
 		locale = QString(QLocale::system().name()).left(2);
 		if (locale.length() < 2) locale = "en";
 	}
-
-	QString txsTranslationFile = findResourceFile("texstudio_" + locale + ".qm");
-
+	QString txsSourceFile = "texstudio_" + locale + ".qm";
+	QString txsTranslationFile = findResourceFile(txsSourceFile);
     if (txsTranslationFile.isEmpty()) {
-        txsTranslationFile = findResourceFile("translation/texstudio_" + locale + ".qm");
+        txsSourceFile = "translation/texstudio_" + locale + ".qm";
+        txsTranslationFile = findResourceFile(txsSourceFile);
     }
+    QString qtSourceFile = "qt_" + locale + ".qm";
+    QString qtTranslationFile = findResourceFile(qtSourceFile);
     bool result0=appTranslator->load(txsTranslationFile);
-    bool result1=basicTranslator->load(findResourceFile("qt_" + locale + ".qm"));
-    if(locale!="en" && (!result0 || !result1) ){

```

Comment:
I should rather like to see my change stay ... (locale!="en"...)

Reply:
had a similar idea...